src/cc_helpers.cc

```diff
@@ -1301,78 +1656,47 @@ bool certifier::framework::cc_trust_data::certify_me(const string& host_name, in
 #endif
 
   if (response.status() != "succeeded") {
-    printf("certifier::framework::cc_trust_data::certify_me: Certification failed\n");
+    printf("%s() error, line: %d, Certification failed\n",
+         __func__, __LINE__);
     return false;
   }
 
+  is_certified_ = true;
   // Store the admissions certificate cert or platform rule
-  if (purpose_ == "authentication") {
+  if (owner_->purpose_ == "authentication") {
+    admissions_cert_.assign((char*)response.artifact().data(), response.artifact().size());
 
-    public_auth_key_.set_certificate(response.artifact());
-    private_auth_key_.set_certificate(response.artifact());
+  } else if (owner_->purpose_ == "attestation") {
 
-#ifdef DEBUG
-    printf("\nAfter cert assigned\n");
-    printf("private key:\n");
-    print_key(private_auth_key_);
-    printf("\n");
-    X509* art_cert = X509_new();
-    if (asn1_to_x509(private_auth_key_.certificate(), art_cert)) {
-      printf("Cert:\n");
-      X509_print_fp(stdout, art_cert);
-    }
-    X509_free(art_cert);
-#endif
-    cc_auth_key_initialized_ = true;
-    cc_is_certified_ = true;
-
-  } else if (purpose_ == "attestation") {
-
-    public_service_key_.set_certificate(response.artifact());
-    private_service_key_.set_certificate(response.artifact());
+    // These should be set in cc_trust_data and ONLY for the primary domain
+    // Todo
```

Comment:
Please use UPPER-CASE either TODO or FIXME. Both show up highlighted font by default in, say, vim.

Also we should adopt the practice of tagging the owner who put this FIXME in paranthesis as "(jhmulcb) FIXME ....

Makes it easier to quickly know who put in this FIXME comment.

Reply:
that was a dangling todo, removed.